docs/source/user_guide/guide.rst

```diff
@@ -53,28 +54,31 @@ Next, we will take a look at the building blocks of `fastplotlib` and how they c
 
 **Figure**
 
-The base of any visualization in `fastplotlib` is a `Figure` object. This can be a singular plot or a grid of subplots.
+The starting for creating any visualization in `fastplotlib` is a `Figure` object. This can be a single plot or a grid of subplots.
```

Comment:
```suggestion
The starting point for creating any visualization in `fastplotlib` is a `Figure` object. This can be a single plot or a grid of subplots.
```